### DISCUSSION TITLE
Which model did you use to predict Mgnify90 sequences?

### DISCUSSION THREAD

**Discussion by zkskek321** [facebookresearch/esm#394] [ESM Metagenomic Atlas]
Hi, I'm microbiome researcher in Korea.

There are many versions of ESM-fold and many versions of pre-trained models depending on the number of parameters.

So, I want to know which ESM-fold version and which pre-trained model did you use to predict Mgnify90_highquality_clust30 protein structure uploaded in github below.
(https://github.com/facebookresearch/esm/blob/main/scripts/atlas/README.md)

Thank you so much.

**Comment by tomsercu** [comment]
Hi! There are two esmfold models: `esmfold_v1` and `esmfold_v0`, both trained on top of the 3B LM but `v1` with more recent data. All predicted structures in the ESM Metagenomic Atlas are predicted using `esmfold_v0`.

**Comment by jgoldford** [comment]
adding a related question to this thread -- which pre-trained model (name and size) was used to generate the embeddings for the ESM atlas?

**Comment by ebetica** [comment]
I'm pretty sure it's the 3B model but i don't have access to the original pipeline anymore unfortunately.
